src/db/dataService/mainQueryDataService.js

```diff
@@ -0,0 +1,11 @@
+import knex from "knex";
```

Comment:
to nie jest mainQueryDataService, tylko robisz operacje na tabeli Search więc searchDataService